certified-connectors/TrueDialog SMS/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1762 @@
+                "properties": {
+                  "Target": {
+                    "type": "string",
+                    "description": "The target that has been filtered out",
+                    "title": "Target"
+                  },
+                  "ReasonId": {
+                    "type": "integer",
+                    "format": "int32",
+                    "description": "ID of the reason. 0 - Invalid, 1 - Opted Out, 2 - Duplicate, 3 - Blocked, 4 - Undeliverable, 5 - Single Use",
+                    "title": "ReasonId"
+                  },
+                  "Reason": {
+                    "type": "string",
+                    "description": "Short description of a reason",
+                    "title": "Reason"
+                  }
+                }
+              },
+              "description": "InvalidTargets"
+            }
+          }
+        }
+      },
+      "put": {
+        "responses": {
+          "201": {
+            "description": "Invalid Targets callback created"
+          }
+        },
+        "summary": "Invalid Targets",
+        "description": "This operation triggers a flow if targets have been filtered out during sending SMS",
+        "operationId": "Invalid_Targets",
+        "x-ms-trigger": "single",
+        "parameters": [
+          {
+            "name": "accountId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Use Root Account ID in most cases to subscribe for notifications.",
+            "x-ms-summary": "Account ID for subscription"
+          },
+          {
+            "name": "Content-Type",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "application/json",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "Accept",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "application/json",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "CallbackType": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "CallbackType",
+                  "title": "CallbackType",
+                  "default": 13,
+                  "x-ms-visibility": "internal"
+                },
+                "URL": {
+                  "type": "string",
+                  "description": "URL",
+                  "title": "URL",
+                  "x-ms-notification-url": true,
+                  "x-ms-visibility": "internal"
+                },
+                "Active": {
+                  "type": "boolean",
+                  "description": "Active",
+                  "title": "",
```

Comment:
@TrueDialogDynamics Empty property values can be removed until unless you have a specific reason to keep them.

Reply:
Fixed empty Titles in [this commit](https://github.com/microsoft/PowerPlatformConnectors/pull/1337/commits/25796ca1fe660681d7eddabb91e8614fbc19a1ef)